c/include/libsbp/observation.h

```diff
@@ -612,6 +612,21 @@ coordinate system
 } msg_almanac_glo_t;
 
 
+/** GLONASS SV orbital and frequency slots mapping information
+ *
+ * The message reports mapping information regrding GLONASS SV orbital and
+ * frequency slots. Frequency numbers -7 to 6 have been reported as 1 to 14
```

Comment:
Well... we could indeed reduce it to 28.. but in the ICD the `n` field is 5 bits and its valid range is stated to be all 0-31 values. So, at the expense of +4 bytes, we will never risk to be unable to map something that is transmitted by the satellite. But this was just a thought.. feel free to re-spin if you think it's more appropriate.